wyvern/components/features/feature_store.py

```diff
@@ -117,40 +114,57 @@ async def fetch_features_from_feature_store(
 
         response_json = await response.json()
         feature_names = response_json["metadata"]["feature_names"]
-        feature_name_keys = [
-            feature_name.replace("__", ":", 1) for feature_name in feature_names
+        feature_names = [
+            feature_name.replace("__", ":") for feature_name in feature_names
```

Comment:
I think having the `1` could still be good right? it would prevent turning the `__` in the actual feature name into `:`

Reply:
I think we should add a general rule that no feature name or feature view can contain __  or : in their names.

Reply:
generally agree with having a general rule like that. But I think having the 1 would not cause issues even if the feature_name has a `__`

more context:
https://github.com/wintonzheng/wyvern/pull/186#discussion_r1253197587

Reply:
ok then sg. i totally forgot about that convo